### PR TITLE
changes on fda.py

### PR DESCRIPTION
**Pull request by FurkyT**
Hello,
I was working on with this repo for fda files and I can not extract the images with decode function after that I add a try except and with except part I decode the data with bytes io. And I add some functions to get the other informations in fda file and wrote another .py file with a function that extract bscans images, video, fundus image, gray scale fundus image and a json which contains other informations in fda file.

### CODE REVIEW

File: oct_converter/readers/fda.py [hunk 1]
```diff
@@ -109,11 +234,18 @@ def read_oct_volume(self):
...
+
             for i in range(oct_header.number_slices):
                 size = np.fromstring(f.read(4), dtype=np.int32)[0]
                 raw_slice = f.read(size)
-                slice = decode(raw_slice)
+                try:
+                    slice = decode(raw_slice)
+                except:
+                    image = Image.open(io.BytesIO(raw_slice))
+                    slice = np.asarray(image)
```
Comment (marksgraham) [inline oct_converter/readers/fda.py:245]: I think we can remove decode here and rely directly on your method. It seems to work well and removes our dependency on `pylibjpeg` which probably causes more errors than any library we depend on.

**Review by marksgraham — CHANGES_REQUESTED** [review]
Hi @FurkyT,

This is a great PR, thanks very much. Being able to remove our dependency on pylibjpeg is very nice.

I've got a few changes I think would be good. If you have the time and are willing to implement them then great, but if not thats fine - I'll merge your PR and work on them myself. 

1. If we fully remove the pylibjpeg import, we can remove the dependency from `pyproject.toml`
2. I think we could reduce duplicate code in all the new methods by having a single method that took a field as input (e.g. @PARAM_OBS_02), checks to see if it exists in the chunk_dict, and if so readers it and returns the read header. Each of the new methods could call that method, only passing the header name, and then assign the relevant fields to a dict.

File: oct_converter/readers/fda.py (hunk 1 above)
Comment (FurkyT) [reply oct_converter/readers/fda.py]: I made the changes and remove the pylibjpeg import and try except lines with decode function.

Comment (FurkyT) [reply oct_converter/readers/fda.py]: Hi for the second request I can work on that but I need a little bit more time for it.

**Review by marksgraham — APPROVED** [review]

**Comment by marksgraham** [comment]
Thanks, will merge for now and await any further updates in a further PR

**Comment by FurkyT** [comment]
Okay thanks for the merge I'll try to work on the improvment on my freetimes.

**Comment by FurkyT** [comment]
Hi,
I have a question, will you release new update for python package.

**Comment by marksgraham** [comment]
Hi @FurkyT 
I've now released to PyPi

**Comment by FurkyT** [comment]
Thanks,
I hope I can make the improvments soon.

**Pull request merged**